### DISCUSSION TITLE
Find debug in visual studio code C++

### DISCUSSION THREAD

**Discussion by Divyalovecoding** [microsoft/vscode-cpptools#8172] [Q&A]
Hi I have some issue in order to debug C++ with visual studio code.
I followed this tutorial : https://code.visualstudio.com/docs/languages/cpp

I'm using Visual Studio Code on windows 10.

I set the path of g++/gcc properly in environment variables in both system variables and user variables. 
Also I close it and again run as administrator. 
Also, when I check in command prompt I shows only fatal error. 

![IMG_20210919_002751](https://user-images.githubusercontent.com/90983425/133905960-c963a8d4-2156-474e-95ec-175916c1f09b.jpg)

![IMG_20210919_002825](https://user-images.githubusercontent.com/90983425/133905972-eeaf3399-e99e-42c3-9f11-712fd42c0fe8.jpg)

Could you please help me, it has been at least a week and I still didn't find the issue.

**Comment by sean-mcmanus** [comment]
You should open a folder. Add the file and do F5. You can find more at https://code.visualstudio.com/docs/cpp/config-mingw .
